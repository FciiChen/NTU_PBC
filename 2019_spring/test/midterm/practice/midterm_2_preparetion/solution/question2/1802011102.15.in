sdi1, 8NB,LekeyqE,S6m
QrpB, ENEg,wfqQebI,sMuH
O0uagsXdU, 90QlbJhj,aMUu8E,ubg
nAeDn0SD, YEo,rI6LoTCH,vNHgd7q
yWgcY2, hIkzECizE,Dk726h4z,BlZVF85
aOZUmwNz, t72bAr,LMTUYGJNTa,kQWGw0URhQ
uYZsDrmPPP, KjA4,Cjqpv2WP8,7gKs
ewsVm, WiiZZ,ntX,YMyJddE
tORW, RAs,GDpbwT,f7L7
BREAK
nAeDn0SD ,  20gfWB
uYZsDrmPPP ,  Fzd1o83Qe
aOZUmwNz ,  1g3dygAm5
BREAK
