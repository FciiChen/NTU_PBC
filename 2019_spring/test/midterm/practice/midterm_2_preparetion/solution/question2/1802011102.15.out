sdi1       8NB        LekeyqE    S6m        NULL      
QrpB       ENEg       wfqQebI    sMuH       NULL      
O0uagsXdU  90QlbJhj   aMUu8E     ubg        NULL      
nAeDn0SD   YEo        rI6LoTCH   vNHgd7q    20gfWB    
yWgcY2     hIkzECizE  Dk726h4z   BlZVF85    NULL      
aOZUmwNz   t72bAr     LMTUYGJNTa kQWGw0URhQ 1g3dygAm5 
uYZsDrmPPP KjA4       Cjqpv2WP8  7gKs       Fzd1o83Qe 
ewsVm      WiiZZ      ntX        YMyJddE    NULL      
tORW       RAs        GDpbwT     f7L7       NULL      
